pkg/proto/eth_crypto.go

```diff
@@ -52,6 +52,19 @@ type EthereumSignature struct {
 	sig [ethereumSignatureLength]byte
 }
 
+func NewEthereumSignatureFromVRS(v byte, R, S *big.Int) (EthereumSignature, error) {
```

Comment:
Bad naming for `R` and `S`. Please, make them lowercase.

Reply:
Fixed